seed/static/seed/js/controllers/members_controller.js

```diff
@@ -109,6 +109,15 @@ angular.module('BE.seed.controller.members', [])
         });
       };
 
+      /**
+       * reset_all_passwords triggers a reset password email for all users
+       */
+       $scope.reset_all_passwords = function (confirm_message = "Really reset all passwords?") {
+         if (confirm(confirm_message)) {
+           organization_service.reset_all_passwords($scope.org.id);
```

Comment:
I think we should log the current user out (route to `/accounts/logout/`) b/c if they try to do anything else it'll error until they reset their password.

We should also notify them before the click "OK", that it will log them out as well.

Reply:
Added a warning and redirected to login once accepted.